strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/nuget/NugetArtifactControllerTest.java

```diff
@@ -402,10 +393,10 @@ public void testLastVersionPackageSearch(@NugetRepository(storageId = STORAGE_ID
 
         // VERSION 1.0.0
         Path packagePathV1 = packagePaths.get(0);
-        byte[] packageContent = readPackageContent(packagePathV1);
+        
         // Push
         String url = getContextBaseUrl() + "/storages/{storageId}/{repositoryId}/";
-        createPushRequest(packageContent).when()
+        createPushRequest(packagePathV1).when()
                                          .put(url, storageId, repositoryId)
```

Comment:
Cosmetic: Could you please re-indent the chained methods? Thanks! :)

Reply:
Fixed.